include/eve/module/core/decorator/promote.hpp

```diff
@@ -0,0 +1,57 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/overload.hpp>
+
+namespace eve
+{
+template<auto Param> struct diff_;
+//================================================================================================
+//================================================================================================
+// Function decorators mark-up used in function overloads
+struct promote_
+{
+  template<auto N> static constexpr auto combine(decorated<diff_<N>()> const&) noexcept
```

Comment:
diff is not even there anymore in EVE

Reply:
this is still not needed

Reply:
i just copied pedantic et là j'ai enlevé la ligne

Reply:
You removed the whole decorator, that is not the issue.